content/docs/command-reference/diff.md

```diff
@@ -33,6 +33,14 @@ sections below for more details.
 `dvc diff` does not have an effect when the repository is not tracked by Git,
 for example when `dvc init` was used with the `--no-scm` option.
 
+By default, when comparing the current workspace to a Git commit, `dvc diff`
+includes a `not in cache` status for DVC-tracked files and directories which are
+neither in the workspace nor the local cache (indicating that they most likely
+need to be `dvc pull`ed into the workspace). The `--hide-missing` option can be
+used to suppress this status and only print files and directories which have
+been expliclity added, modified or deleted in the workspace. This status will be
+automatically suppressed when comparing two Git commits.
```

Comment:
We can just explain the option in the Options section I think. Like `--show-md` which isn't mentioned in the Description. These are uncommon options, I think.

Reply:
I see Ivan already mentioned this in https://github.com/iterative/dvc.org/pull/1758#pullrequestreview-484504526 above. Resolving here.